clients/drcachesim/scheduler/scheduler_dynamic.cpp

```diff
@@ -113,7 +113,7 @@ scheduler_dynamic_tmpl_t<RecordType, ReaderType>::set_initial_schedule(
     }
     // Now assign round-robin to the outputs.  We have to obey bindings here: we
     // just take the first.  This isn't guaranteed to be perfect if there are
-    // many bindings, but we run a rebalancing afterward.
+    // many bindings (or output limits), but we run a rebalancing afterward.
```

Comment:
This needs more explanation: what is the challenge in considering output limits (or "many bindings" (not sure how that's different than "bindings" that we claim to obey above) for that matter) here?